src/explore-education-statistics-admin/src/services/importStatusBauService.ts

```diff
@@ -0,0 +1,21 @@
+import client from '@admin/services/utils/service';
+
+export interface ImportStatusBau {
+  subjectTitle: string;
+  subjectId: string;
+  publicationId: string;
+  releaseId: string;
+  dataFileName: string;
+  metaFileName: string;
+  numberOfRows: number;
+  status: number;
+  stagePercentageComplete: number;
+}
+
+const importStatusBauService = {
```

Comment:
`importStatusService`

Reply:
I think this could also lead to backend/frontend confusion as on the previous comment.